islandora_solr_config/includes/atom_results.inc

```diff
@@ -0,0 +1,248 @@
+<?php
+
+/**
+ * @file
+ * Contains methods to search solr and display results.  depends on
+ * Apache_Solr_Php client.
+ */
+
+/**
+ * Extention of IslandoraSolrResults for templating purposes.
+ * This overrides the displayResults function to provide an alternate display
+ * type.
+ */
+
+class IslandoraSolrResultsAtom extends IslandoraSolrResults {
+
+  /**
+   * Outputs results basically in the normal way.
+   *
+   * Thumbnails pulled from the Fedora repository.
+   *
+   * @param object $islandora_solr_query
+   *   A solr query object.
+   *
+   * @return string
+   *   html output for the resultset. Note: we currently create this
```

Comment:
Comment doesn't seem correct. This isn't HTML.

Reply:
Yeah, that's just a leftover from the rss_results.inc file which I hacked this from. Will change to "xml".